amiqus/models/step.py

```diff
@@ -17,10 +21,27 @@ class Step(models.Model):
 
     id = models.AutoField(primary_key=True)
 
+    amiqus_id = models.CharField(
+        "Amiqus ID",
+        max_length=40,
+        help_text=_(
+            "The id returned from the Amiqus API. These should be unique on "
+            "the record."
+        ),
+        null=True,
+    )
+
     record = models.ForeignKey(Record, on_delete=models.CASCADE, related_name="steps")
     amiqus_check = models.OneToOneField(
         Check, on_delete=models.CASCADE, null=True, blank=True, related_name="step"
     )
     form = models.OneToOneField(
         Form, on_delete=models.CASCADE, null=True, blank=True, related_name="step"
     )
+
+    def __str__(self) -> str:
+        return f"Step number {self.amiqus_id} on record {self.record}"
```

Comment:
I'm pretty sure that doing `self.record` will still cause a table jump. This should probably be:

```suggestion
        return f"Step number {self.amiqus_id} on record #{self.record_id}"
```